client/src/pages/SearchMentor.js

```diff
@@ -1,17 +1,46 @@
 import PropTypes from 'prop-types';
 import MentorsCard from '../components/MentorsCard';
+import Searchbar from '../components/Searchbar';
 
 
-export default function SearchMentor({ open, mentors, addToFavouriteMentor, favourites }) {
+export default function SearchMentor({ open, mentors, addToFavouriteMentor, favourites}) {
+
+const params = new URLSearchParams(window.location.search);
+const competenceInput = params.get("competence");
+const buzzwordsInput = params.get("buzzwords");
```

Comment:
I would recommend to rethink this approach. Currently this works because your form <Searchbar /> is being submitted which causes a page reload. We don't want this (as we follow a single page application approach in react). Maybe you find a different approach to filter your mentors by competence or buzzwords. Not sure if maybe also the react router has some options that could help you? :) Just food for thought 😸